### PR TITLE
fix pad2d example code

### PR DESCRIPTION
**Pull request by littletomatodonkey**
### PR types
Others

### PR changes
Docs

### Describe
Improve pad2d docs

### CODE REVIEW

**Review by willthefrog — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -9340,9 +9340,35 @@ def pad2d(input,
...
         .. code-block:: python
 
-            import paddle.fluid as fluid
-            data = fluid.data(name='data', shape=[None, 3, 32, 32], dtype='float32')
-            result = fluid.layers.pad2d(input=data, paddings=[0, 1, 2, 3], mode='reflect')
+            import numpy as np
+            import paddle
+            import paddle.nn.functional as F
+
+            paddle.disable_static()
```
Comment (shippingwang) [inline python/paddle/fluid/layers/nn.py]: unnecessary disabled now?

Comment (littletomatodonkey) [reply python/paddle/fluid/layers/nn.py]: deleted, thx

**Review by shippingwang — DISMISSED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by ceci3 — APPROVED** [review]

**Pull request merged**